src/util/convert_bootstrap.py

```diff
@@ -14,9 +14,12 @@
 except ImportError:
     from distutils.version import StrictVersion as Version  # N.B. deprecated, will be removed in py3.12
 
+import warnings
 
 _logger = logging.getLogger(__name__)
 
+# Filter out deprecation warnings for the import from distutils.version
+warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
1. put it in the `ImportError` part
2. only ignore the DeprecationWarning about the `Version` class. Don't ignore all warnings.

Reply:
@thni-odoo use [`message`](https://docs.python.org/3.6/library/warnings.html#the-warnings-filter) parameter
```suggestion
# Filter out deprecation warnings for the import from distutils.version
warnings.filterwarnings("ignore", message=r"DeprecationWarning: distutils Version classes are deprecated\. Use packaging\.version instead", category=DeprecationWarning)
```

Reply:
Thanks for the suggestion! Changes is done